README.md

```diff
@@ -101,6 +101,58 @@ add `priority` label when you start backing us :smile:
 
 See: [docs/deployment.md](docs/deployment.md)
 
+# Development
+
+> This short guide is meant to give a guide as to how github apps are made and basic steps to reproduce the OpenCollective Bot should you wish to make futher changes.  
+
+ ## Installation
+
+Feel free to clone or download this project using the **git clone** and install using **Yarn install** or **Npm install** to install required dependencies
+
+**WebHooks** are crucial to **Github Apps**. Hence, you can use [Ngrok](https://ngrok.com) , [Smee](https://smee.io) or other comfortable options .
+
+> **Ngrok** is however used in this guide . 
+
+### Running Ngrok
+
+> After making all necessary local Ngrok installations found here [Ngrok](https://ngrok.com)
+
+- Run `Ngrok http <{localhost port}>` from your downloaded `Ngrok.exe` to create a tunnel in your terminal.
+- Take note of the **Web Interface** generated as it would be needed when registering the App on github.
+
+### Registering App
+
+> Follow steps listed at [Github](https://developer.github.com/apps/building-github-apps/creating-a-github-app/) to register your bot. 
+ Use the **Web Interface** as **Webhook Url** when registering the app on Github.    
+
+#### Credentials
+
+After registering the app, take note of the following listed details shown in the about page 
+
+- App ID
+- Client ID
+- Client Secret
+
+> Using a `.env` file in this project is highly recommended to store your credentials  
+
+- Create a new `.env` file at the root folder and copy the code below into it.
+  
+```js
+ APP_ID = <your app client id>
+ WEBHOOK_SECRET = <your app client secret>
+ PRIVATE_KEY = <your app id>
+ SENTRY_DSN = <base64 encoded contents of your pem file>
```

Comment:
The description is inaccurate. I would suggest to skip `SENTRY_DSN`, it's optional and explaining it would add complexity.